presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -356,6 +357,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         PUSHDOWN_FILTER_ENABLED,
                         "Experimental: enable complex filter pushdown",
                         false,
+                        false),
+                integerProperty(
+                        VIRTUAL_BUCKET_COUNT,
+                        "Number of virtual bucket assigned for unbucketed tables",
+                        1,
```

Comment:
My original intention is to treat `bucket count = 1` as unbucketed because 1 bucket is effectively unbucketed. But I think it also makes sense to treat `bucket count = 0` as unbucketed and treat `bucket count = 1` as a bucketed table with single bucket.